lib/error_module.rb

```diff
@@ -0,0 +1,23 @@
+module Error_module
```

Comment:
The convention or naming classes and modules is CameCase so the name should be `ErrorModule´